src/Peachpie.Runtime/Reflection/PhpRoutineInfo.cs

```diff
@@ -290,6 +294,30 @@ public PhpMethodInfoWithBoundType(int index, string name, MethodInfo[] methods,
 
         #endregion
 
+        #region BoundTargetCallable
```

Comment:
remove, move to `PhpCallback`

Reply:
OK, updated in 2828173f318c841df8a368d1e4817d6ec702e2b6.